plugins/balloonpanel/plugin.js

```diff
@@ -383,8 +386,19 @@
 				left: 'right'
 			};
 
-			return function( element, focusElement ) {
-				this.show();
+			return function( element, options ) {
+				if ( options instanceof CKEDITOR.dom.element || typeof options === 'boolean' || typeof options === 'undefined' ) {
```

Comment:
Still this construction will allow `null` to slip by, and as `null` gets passed to `CKEDITOR.tools.extend` we get an exception. I have added TC for that in 70c565c54.

The simplest solution here is just change `typeof options === 'boolean' || typeof options === 'undefined'` into a `!options' and that will take care of falsy values.